web-ui/src/main/resources/catalog/components/admin/harvester/HarvesterDirective.js

```diff
@@ -130,6 +131,82 @@
     }
   ]);
 
+  /**
+   * Display translation service configuration
+   */
+  module.directive("gnHarvesterTranslateContentConfigurator", [
```

Comment:
With this refactor the language picker in the harvester doesn't work. Clicking in the field to select a language:

```
UtilityDirective.js:1029 Uncaught TypeError: Cannot read properties of null (reading 'all')
    at Dataset.allOrSearchFn [as source] (UtilityDirective.js:1029:33)
    at Dataset.update (typeahead.bundle.js?v=e6c3a99c1fbe6adea82e4e57eceddf502bff2fe9:1708:22)
    at updateDataset (typeahead.bundle.js?v=e6c3a99c1fbe6adea82e4e57eceddf502bff2fe9:1889:29)
```

Reply:
It works in debug mode, but not in the production mode